package.json

```diff
@@ -20,29 +19,29 @@
   },
   "homepage": "https://github.com/canopytax/cp-select",
   "devDependencies": {
-    "angular-mocks": "^1.5.2",
-    "autoprefixer-loader": "^3.2.0",
-    "babel-core": "^6.7.4",
-    "babel-loader": "^6.2.4",
-    "css-loader": "^0.23.1",
-    "html-loader": "^0.4.3",
-    "jasmine-core": "^2.4.1",
-    "karma": "^0.13.22",
-    "karma-chrome-launcher": "^0.2.3",
-    "karma-jasmine": "^0.3.8",
-    "karma-phantomjs-launcher": "^1.0.0",
-    "ngmin": "^0.5.0",
-    "ngmin-webpack-plugin": "^0.1.3",
-    "node-static": "^0.7.7",
-    "phantomjs-prebuilt": "^2.1.6",
-    "style-loader": "^0.13.1",
-    "webpack": "^1.12.14"
+    "angular-mocks": "^1.6.8",
+    "babel-cli": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-angularjs-annotate": "^0.8.2",
+    "babel-preset-env": "^1.6.1",
+    "css-loader": "^0.28.9",
+    "html-loader": "^0.5.5",
+    "jasmine-core": "^2.9.1",
+    "karma": "^2.0.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-jasmine": "^1.1.1",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "node-static": "^0.7.10",
+    "phantomjs-prebuilt": "^2.1.16",
+    "postcss-loader": "^2.0.10",
+    "style-loader": "^0.19.1",
+    "webpack": "^3.10.0"
   },
   "dependencies": {
-    "angular": "^1.5.2",
-    "babel": "^6.5.2",
-    "jquery": "^2.2.2",
-    "lodash": "^4.6.1"
+    "angular": "^1.6.8",
+    "babel-cli": "^6.26.0",
+    "jquery": "^3.3.1",
```

Comment:
THE whole reason to update this project. Updating jQuery to eliminate a known vulnerability in the older version